metadata/contributors.json

```diff
@@ -2490,6 +2490,16 @@
          "efidler"
       ]
    },
+   {
+      "emails" : [
+         "fantasai@apple.com",
+         "fantasai.bugs@inkedblade.net"
```

Comment:
I think that this needs to come first, so that Bugzilla autofill uses it (or we could change your Bugzilla email address instead).